package.json

```diff
@@ -1,108 +1,43 @@
 {
-  "name": "fd-vue-webapp",
-  "version": "1.0.0",
-  "description": "A Vue.js web application for Freedomotic",
-  "author": "Freedomotic Team",
+  "name": "freedomotic",
+  "version": "0.0.1",
+  "description": "A Quasar Framework app",
+  "productName": "Freedomotic Web App",
+  "cordovaId": "org.cordova.quasar.app",
+  "author": "atanda rasheed <atandarash@gmail.com>",
   "private": true,
   "scripts": {
-    "dev": "node build/dev-server.js",
-    "start": "node server.js",
-    "heroku-postbuild": "npm install --only=dev --no-shrinkwrap && npm run build",
-    "build": "node build/build.js",
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
-    "e2e": "node test/e2e/runner.js",
-    "test": "npm run unit && npm run e2e",
-    "lint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs"
+    "lint": "eslint --ext .js,.vue src",
+    "test": "echo \"No test specified\" && exit 0"
```

Comment:
why are you removing tests?

Reply:
Those scripts can be updated, that's the default **package.json** created when the quasar-cli is used to generate a project